### ISSUE TITLE
repost of previous days artwork

### ISSUE THREAD

**Issue by theyellingmute** [spslater/twitter2reddit#2]
Today's post simply reposted the last artwork from yesterday. 

Im not a programmer but I imagine it might just be checking the last post from divas twitter that matches the description and then posts it with the number increasing by 1? 

Now with it being every other day, may need to be adjusted.

**Comment by spslater** [comment]
Yep! That's exactly what happened, thanks for the heads up. We're trying to figure out what the new schedule is and how best to get the bot to check for them. It's gonna be manually posted for a couple days until I am able to get this fixed.
